.misc/deploy.pypi.sh

```diff
@@ -4,18 +4,22 @@
 set -e
 
 # Get environment variables, readily decrypted by rultor
-source ../rultor_secrets.sh
+#source ../rultor_secrets.sh
 
 # Ship it!
 echo "Uploading coala to pypi"
 pip3 install twine wheel
 python3 setup.py sdist bdist_wheel
 # Upload one by one to avoid timeout
-# Don't block merging because pypi is in error 500 mood again
-set +e
-twine upload dist/*.whl -u "$PYPIUSER" -p "$PYPIPW"
-twine upload dist/*.tar.gz -u "$PYPIUSER" -p "$PYPIPW"
-set -e
+twine upload dist/* -u "$PYPIUSER" -p "$PYPIPW" 2>&1 | tee twine_output.txt
```

Comment:
1. Can `twine upload dist/*.whl` and `twine upload dist/*.tar.gz` be safely replaced by `twine upload dist/*`?
2. Up to you: Maybe use a variable for `twine_output.txt` as you use it three times^^